statusnet/statusnet.php

```diff
@@ -757,12 +760,15 @@ function statusnet_cron(App $a, array &$b)
 
 	$abandon_limit = date(DateTimeFormat::MYSQL, time() - $abandon_days * 86400);
 
+	/// @TODO Use PConfig::get() here maybe?
```

Comment:
No, like @MrPetovan said above, we are are fetching all users here who have this setting activated.